src/Estimators/src/BaseEstimatorFromFootIMU.cpp

```diff
@@ -111,47 +107,80 @@ bool BaseEstimatorFromFootIMU::initialize(
         return true;
     };
 
-    // Base link of the robot (whose pose must be estimated)
-    std::string baseFrameName;
-    bool ok = populateParameter(ptr->getGroup("MODEL_INFO"), "base_frame", baseFrameName);
-
-    // Frame associated to the foot of the robot (whose orientation is measured)
-    ok = populateParameter(ptr->getGroup("MODEL_INFO"), "foot_frame", m_footFrameName);
-
-    ok = ok && populateParameter(ptr, "foot_width_in_m", m_footWidth);
-    ok = ok && populateParameter(ptr, "foot_length_in_m", m_footLength);
+    auto baseEstimatorPtr = ptr->getGroup("BASE_ESTIMATOR").lock();
+    if (baseEstimatorPtr == nullptr)
+    {
+        log()->error("{} Invalid parameter handler for the group 'BASE_ESTIMATOR'", logPrefix);
+        return false;
+    }
 
-    // Set the 4 foot vertices in World reference frame [dimensions in meters]
-    m_cornersInLocalFrame.emplace_back(+m_footWidth / 2, +m_footLength / 2, 0);
-    m_cornersInLocalFrame.emplace_back(-m_footWidth / 2, +m_footLength / 2, 0);
-    m_cornersInLocalFrame.emplace_back(-m_footWidth / 2, -m_footLength / 2, 0);
-    m_cornersInLocalFrame.emplace_back(+m_footWidth / 2, -m_footLength / 2, 0);
+    // Frame associated to the base of the robot (whose pose is estimated)
+    bool ok = populateParameter(baseEstimatorPtr->getGroup("MODEL_INFO"),
```

Comment:
Consider storing the output of ``baseEstimatorPtr->getGroup("MODEL_INFO")`` since you are using it multiple times below